coalib/output/ConsoleInteraction.py

```diff
@@ -97,22 +97,24 @@ def highlight_text(no_color, text, style, lexer=TextLexer()):
 
 
 def color_letter(console_printer, line):
-    x = -1
-    y = -1
-    letter = ''
-    for i, l in enumerate(line, 0):
-        if line[i] == '(':
-            x = i
-        if line[i] == ')':
-            y = i
-        if l.isupper() and x != -1:
-            letter = l
+    x = line.find('(')
+    if x == -1:
+        letter = ''
+        y = x + 1
+    else:
+        letter = line[x + 1]
+        y = x + 2
+    warn = line.rfind('[')
+    if warn == 0:
```

Comment:
Shouldn't it be `-1` if not found?

Reply:
No, notice `[` is always the first character when you are asked for an action.